src/Microsoft.AspNet.Http.Features/Authentication/AuthenticateContext.cs

```diff
@@ -41,5 +41,12 @@ public virtual void NotAuthenticated()
         {
             Accepted = true;
         }
+
+        public bool IsRequestCompleted { get; private set; }
```

Comment:
Yeah I guess authenticate is special, maybe we don't need this on the authenticate context since most of the handlers have specialized implementations anyways, and its mostly creating the ticket/error, as opposed to actually finishing the response/request